docs/Populate.md

```diff
@@ -0,0 +1,118 @@
+# Populate Exchange
+
+`populate` is an exchange for auto-populating fields in your mutations.
+
+## How to use
+
+As with any exchange, add this to your client (before your cache).
+
+```tsx
+import { populateExchange } from '@urql/exchange-graphcache';
+
+const client = createClient({
+  // ...
+  exchanges: [dedupExchange, populateExchange, cacheExchange, fetchExchange],
+})
+```
+
+## Example usage
+
+Consider the following queries which have been requested
+
+
+```graphql
+# Query 1
+{
+  todos {
+    id
+    name
+  }
+} 
+
+# Query 2
+{
+  todos {
+    id
+    createdAt
+  }
+}
+```
+
+Now consider the following mutation which returns a todo
+
+```graphql
+# Without populate
+mutation addTodo(id: ID!) {
+  addTodo(id: $id) {
+    id        # To update Query 1 & 2
+    name      # To update Query 1
+    createdAt # To update Query 2
+  }
+}
+```
+
+By using the `populate` exchange, you no longer need to manually specify the selection set required to update your other queries.
```

Comment:
```suggestion
By using the `populateExchange`, you no longer need to manually specify the selection set required to update your other queries. Instead you can just add the `@populate` directive.
```

Reply:
I hear your suggestion and I counter you 

> By using `populateExchange`
or
> By using the _Populate_ exchange

Reply:
I think `populateExchange` being the export we should use that as long as that's what it's called when you import it 🤷‍♀ It's the same for `cacheExchange` being our export, but still not being referred to as "Cache Exchange" or "Graphcache" in our docs to avoid confusing it with the entire package. Both works though as long as it's consistent.

One thing we should be careful about is to do say "`@populate` directive" consistently instead of "adding populate" or "populate directive" so it can be spotted easily by someone just skimming the docs.

Reply:
Sure lets go with that, I'll just remove "the"

```suggestion
By using `populateExchange`, you no longer need to manually specify the selection set required to update your other queries. Instead you can just add the `@populate` directive.
```